.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
-# Description
+## Description
 
 Fixes #issue.
 
-# Test Plan
+## Test Plan
 
+- [ ] Confirm that the CI build is successful
```

Comment:
We might as well include the CI test job. Could just say "Confirm CI jobs are successful"